docs/resources/cluster.md

```diff
@@ -74,6 +77,59 @@ resource "rke_cluster" "cluster" {
 
 **Note** Once the RKE cluster is restored, `rke_cluster.restore.restore` will be set to `false` to force tf diff on next apply until user set `rke_cluster.restore.restore = false` on tf file
 
+Provision RKE cluster with pre-defined PSACT. This is only supported on Kubernetes 1.26 clusters.
+
+```hcl
+resource "rke_cluster" "cluster" {
+  cluster_name = "foo"
+  nodes {
+    address = "1.2.3.4"
+    user    = "ubuntu"
+    role    = ["controlplane", "worker", "etcd"]
+    ssh_key = file("~/.ssh/id_rsa")
+  }
+  authorization {
+    mode = "rbac"
+  }
+  services {
+    kube_api {
+      pod_security_configuration = <VALUE> # privileged or restricted
+    }
+  }
+  upgrade_strategy {
+    drain = true
+    max_unavailable_worker = "20%"
+  }
+}
+```
+
+Provision RKE cluster with custom PSACT. This is only supported on Kubernetes 1.26 clusters.
+
+```hcl
+resource "rke_cluster" "cluster" {
+  cluster_name = "foo"
+  nodes {
+    address = "1.2.3.4"
+    user    = "ubuntu"
+    role    = ["controlplane", "worker", "etcd"]
+    ssh_key = file("~/.ssh/id_rsa")
+  }
+  authorization {
+    mode = "rbac"
+  }
+  kube_api {
+    extra_args = {
+      admission-control-config-file: "<CONTAINER_PATH>/admisison.yaml" // path in kube API server container
+    }
+    extra_binds = ["<NODE_PATH>:<CONTAINER_PATH>"]
+  }
+  upgrade_strategy {
+    drain = true
+    max_unavailable_worker = "20%"
+  }
+}
+```
+
```

Comment:
Can you add another example of using the `services.kube-api.admission_configuration` field?